themes/bootstrap3/templates/RecordDriver/DefaultRecord/core-fields.phtml

```diff
@@ -0,0 +1,12 @@
+<?php
+  $formatter = $this->recordDataFormatter($this->driver);
+  $coreFields = $formatter->getData($formatter->getDefaults($this->defaults));
```

Comment:
Would it make sense to use `$this->defaults ?? 'core'` here so we default to reasonable defaults?